arm-customer-insights/2017-01-01/swagger/customer-insights.json

```diff
@@ -4765,17 +4773,17 @@
           "type": "string",
           "description": "The primary participant property name for an interaction ,This is used to logically represent the agent of the interaction, Specify the participant name here from ParticipantName."
         },
-        "dataSources": {
```

Comment:
Hi, this is intentional and it's OK because the 2017-01-01 version never get released before. Last time when we update this swagger, this part was a work-in-progress. Now that it's done and it should look like this (when you call this API with 2017-01-01 version, this is what the returned schema looks like)